RecoLocalCalo/EcalRecProducers/plugins/EcalUncalibRecHitWorkerMultiFit.cc

```diff
@@ -250,7 +250,6 @@ double EcalUncalibRecHitWorkerMultiFit::timeCorrection(
 }
 
 
-
 bool
 EcalUncalibRecHitWorkerMultiFit::run( const edm::Event & evt,
                 const EcalDigiCollection::const_iterator & itdg,
```

Comment:
in case there's confusion. 
What I mean to suggest here is to change the current old per-digi method to
```
 bool
 EcalUncalibRecHitWorkerMultiFit::run( const edm::Event & evt,
                 const EcalDigiCollection::const_iterator & itdg, 
 EcalUncalibratedRecHitCollection & result ){
EcalDigiCollection tempCollection(itdg);
run(evt, tempCollection, output);
```
so that this is available in case someone somewhere wants to run this method for EcalUncalibRecHitWorkerMultiFit.
The other less "nice" alternative is

```
bool
 EcalUncalibRecHitWorkerMultiFit::run( const edm::Event & evt,
                 const EcalDigiCollection::const_iterator & itdg, 
 EcalUncalibratedRecHitCollection & result ){
throw cms::Exception("LogicError") <<"never call this method for EcalUncalibRecHitWorkerMultiFit";
}
```